From 4eb9577004521127daa39444ae35279e8f07eccd Mon Sep 17 00:00:00 2001
From: Paul Mundt <lethal@linux-sh.org>
Date: Sat, 19 May 2012 15:11:45 +0900
Subject: irqdomain: Kill off duplicate definitions.

Presently irqdomain.h has duplicate definitions for irq_find_host() and
irq_set_default_host(), presumably from merge damage. Kill off the
duplicates.

Signed-off-by: Paul Mundt <lethal@linux-sh.org>
Signed-off-by: Grant Likely <grant.likely@secretlab.ca>
(cherry picked from commit cb5557bec9f14d05204a9014ae1b23aca8b04f1d)

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 include/linux/irqdomain.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/include/linux/irqdomain.h b/include/linux/irqdomain.h
index a796dbf..5abb533 100644
--- a/include/linux/irqdomain.h
+++ b/include/linux/irqdomain.h
@@ -144,9 +144,6 @@ static inline struct irq_domain *irq_domain_add_legacy_isa(
 
 extern void irq_domain_remove(struct irq_domain *host);
 
-extern struct irq_domain *irq_find_host(struct device_node *node);
-extern void irq_set_default_host(struct irq_domain *host);
-
 extern unsigned int irq_create_mapping(struct irq_domain *host,
 				       irq_hw_number_t hwirq);
 extern void irq_dispose_mapping(unsigned int virq);
-- 
1.8.0.197.g5a90748

